fed/proxy/barriers.py

```diff
@@ -235,15 +202,33 @@ async def send(
         dest_addr = self._cluster[dest_party]['address']
         dest_party_grpc_config = self.setup_grpc_config(dest_party)
         try:
```

Comment:
I am not sure whether this will be called concurrently. Should we init stubs in `__init__` to ensure that it is only initialized once.